vpr/src/pack/cluster.cpp

```diff
@@ -788,6 +820,23 @@ std::map<t_logical_block_type_ptr, size_t> do_clustering(const t_packer_opts& pa
     return num_used_type_instances;
 }
 
+static void print_pack_status(int num_clb,
+                              int tot_num_molecules,
+                              int num_molecules_processed,
+                              std::map<t_logical_block_type_ptr, size_t> clb_types) {
+    //print an update every 10 clusters
+    if (num_clb % 50 == 0) {
```

Comment:
calculate this as a percent of molecules, put a const float that will be the percentage of how often to print, will have num_molecules so can just multiply and see when to print

Reply:
might need another variable that is molecules since last print

Reply:
reset to 0 so you can have around every 4% message printing

Reply:
and pass in by reference